staging/src/k8s.io/legacy-cloud-providers/azure/auth/azure_auth.go

```diff
@@ -132,6 +142,78 @@ func GetServicePrincipalToken(config *AzureAuthConfig, env *azure.Environment) (
 	return nil, ErrorNoAuth
 }
 
+// GetMultiTenantServicePrincipalToken creates a new multi-tenant service principal token based on the configuration
```

Comment:
Could you add a comment why two tokens are added?

Reply:
Thank you. Updated.